src/pages/tests/example-two.hbs

```diff
@@ -0,0 +1,322 @@
+--- title: Page Example Two layout: blank ---
```

Comment:
I see this being rendered as text in the top of the page. Was that what you wanted it to do?